tests/unit/models/util/LoadShow.spec.ts

```diff
@@ -0,0 +1,46 @@
+import Show from '@/models/Show';
+import { LoadShow } from '@/models/util/LoadShow';
+
+describe('models/util/LoadShow', () => {
+
+  describe('testLoad', () => {
+
+    it('loading show', () => {
+      // I tried useing expect().toThrow, but it didn't work.  Rolling myself
+      try {
+        LoadShow(new ArrayBuffer(8));
+      } catch (e) {
+        expect(e).toBe('file is not a CalChart show file.');
+      }
+    });
+    it('simple show', () => {
+      /* eslint-disable max-len */
```

Comment:
Similar suggestion for others that only cover one line
```suggestion
      // eslint-disable-next-line max-len
```

Reply:
Oh, that's awesome!  Done!